knimin/lib/data_access.py

```diff
@@ -791,7 +791,11 @@ def format_survey_data(self, md, external_surveys=None, full=False):  # noqa
             # Get rid of columns not wanted for pulldown
             if not full:
                 for col in ebi_remove:
-                    del md[1][barcode][col]
+                    try:
+                        del md[1][barcode][col]
+                    except:
+                        # Column doesn't exist already, so no removal needed
```

Comment:
This is just for debugging purposes in the future.

What do you think about adding a warning, indicating that a column that doesn't exist was attempted to be deleted?